.github/workflows/e2e-test.yml

```
@@ -39,16 +39,29 @@ jobs:
 
-      - name: Setup e2e test
-        run: |
-          npm install -g corepack@latest
-          corepack enable
-          pnpm install --frozen-lockfile
-          pnpm playwright install
+      - name: Setup
+        uses: ./tooling/github/setup
+
+      - name: Cache Playwright browsers
+        uses: actions/cache@v3
```

actionlint (1.7.8)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)